apps/gp2-frontend/src/users/UserList.tsx

```diff
@@ -36,6 +60,11 @@ const UserList: React.FC<UserListProps> = ({ displayFilters = false }) => {
         currentPageIndex={currentPage}
         renderPageHref={renderPageHref}
         onFiltersClick={onFiltersClick}
+        onExportClick={exportUsers}
+        isAdministrator={isAdministrator}
+        searchQuery=""
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        onSearchQueryChange={() => {}}
```

Comment:
Are these for when we implement the search?

Reply:
yeah, I've added it so it looks right, but it's still not used